### PR TITLE
.lang: Ensure rawtarget actually has an {ok}

### PR DESCRIPTION
**Pull request by dwaynebailey**

### CODE REVIEW

**Comment by unho** [comment]
lgtm

File: translate/storage/test_mozilla_lang.py [hunk 1]
```diff
@@ -64,6 +64,29 @@ def test_comments(self):
             "# TAG: goodbye"
             in unit.getnotes(origin="developer").split("\n"))
 
+    def test_copy_target(self):
+        """Validate that self.rawtarget does not break a valid translation.
+
+        The rawtarget is used to preserve strange format anomolies related to
```
Comment (unho) [inline translate/storage/test_mozilla_lang.py]: Typo: anomolies

File: translate/storage/test_mozilla_lang.py [hunk 2]
```diff
@@ -64,6 +64,29 @@ def test_comments(self):
...
+        {ok}.  But when units got translated t sometimes cause issues.  This
```
Comment (unho) [inline translate/storage/test_mozilla_lang.py]: Typo: t

File: translate/storage/test_mozilla_lang.py [hunk 3]
```diff
@@ -64,6 +64,29 @@ def test_comments(self):
...
+        checks that translation actually results in rawformat being ignored.
```
Comment (unho) [inline translate/storage/test_mozilla_lang.py]: rawformat?

File: translate/storage/test_mozilla_lang.py [hunk 4]
```diff
@@ -64,6 +64,31 @@ def test_comments(self):
...
+        unit.rawtarget = "Device"
+        assert not unit.istranslated()
+        assert str(unit) == ";Device\nDevice"
+
+        other = self.UnitClass("Device")
+        other.target = "Device"
+        assert other.istranslated()
+        assert str(other) == ";Device\nDevice {ok}"
+
+        unit.target = other.target
```
Comment (unho) [inline translate/storage/test_mozilla_lang.py]: Nit: I would use `unit.target = unit.source`

Comment (dwaynebailey) [inline translate/storage/test_mozilla_lang.py]: That's not we're testing

Comment (unho) [inline translate/storage/test_mozilla_lang.py]: I see. I thought you had an assert for `other.target`

**Comment by unho** [comment]
lgtm

**Pull request closed without merging**